### PR TITLE
Add 0.6 release announce

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

**Comment by qiujian16** [comment]
/hold

still waiting for more input.

File: content/en/community/releases.md [hunk 1]
```diff
@@ -5,7 +5,30 @@ weight: -20
...
 Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.5.0`. Continue reading to view upcoming releases:
 
-## `v0.6.0`, on 21st, Janurary 2022
+## `v0.6.0`, on 21st, January 2022
+
+Open Cluster Management team is proud to annouce the release of OCM v0.6.0! We made many enhancements on core components and introduce some new addons.
+
+- [First release of cluster-proxy addon](https://github.com/open-cluster-management-io/cluster-proxy)
+- [First release of managed-serviceaccount addon](https://github.com/open-cluster-management-io/managed-serviceaccount)
+- [Syn status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
```
Comment (mikeshng) [inline content/en/community/releases.md:14]:
```suggestion
- [Sync status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
```

File: content/en/community/releases.md [hunk 2]
```diff
@@ -5,7 +5,30 @@ weight: -20
...
+- [Placement extensible scheduling](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/32-extensiblescheduling)
+- [Helm chart interface for addon framework](https://github.com/open-cluster-management-io/addon-framework/pull/62)
+
+We also add many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhacements on documentation in the website.
```
Comment (mikeshng) [inline content/en/community/releases.md:18]:
```suggestion
We also added many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhancements on documentation in the website.
```

File: content/en/community/releases.md [hunk 3]
```diff
@@ -5,7 +5,30 @@ weight: -20
...
+
+To see details of the changelogs in this release:
+- registration v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
+- work v0.6.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
+- placement v0.3.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.3.0/CHANGELOG/CHANGELOG-v0.3.md)
+- addon-framework v0.2.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.1.0/CHANGELOG/CHANGELOG-v0.2.md)
+- registration-operator v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.6.md)
+- multicloud-operators-subscription v0.6.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-subscription/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
+- cluster-proxy v0.1.3 [changelog]
```
Comment (mikeshng) [inline content/en/community/releases.md:27]:
```suggestion
- cluster-proxy v0.1.3 [repo](https://github.com/open-cluster-management-io/cluster-proxy)
```

**Review by mikeshng — CHANGES_REQUESTED** [review]
Just some minor nits. Thank you for the release summary.

File: content/en/community/releases.md [hunk 4]
```diff
@@ -5,7 +5,30 @@ weight: -20
...
+## `v0.6.0`, on 21st, January 2022
+
+Open Cluster Management team is proud to annouce the release of OCM v0.6.0! We made many enhancements on core components and introduce some new addons.
+
+- [First release of cluster-proxy addon](https://github.com/open-cluster-management-io/cluster-proxy)
+- [First release of managed-serviceaccount addon](https://github.com/open-cluster-management-io/managed-serviceaccount)
+- [Syn status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
+- [Placement extensible scheduling](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/32-extensiblescheduling)
+- [Helm chart interface for addon framework](https://github.com/open-cluster-management-io/addon-framework/pull/62)
+
```
Comment (mikeshng) [inline content/en/community/releases.md:17]:
```suggestion
- [Placement API support for multicloud-operators-subscription](https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/55)

```

File: content/en/community/releases.md [hunk 5]
```diff
@@ -5,7 +5,37 @@ weight: -20
...
+- [Placement API support for multicloud-operators-subscription](https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/55)
+
+We also added many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhancements on documentation in the website.
+
+To see details of the changelogs in this release:
+- registration v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
+- work v0.6.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md)
+- placement v0.3.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.3.0/CHANGELOG/CHANGELOG-v0.3.md)
+- addon-framework v0.2.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.1.0/CHANGELOG/CHANGELOG-v0.2.md)
+- registration-operator v0.6.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.5.0/CHANGELOG/CHANGELOG-v0.6.md)
```
Comment (zhujian7) [inline content/en/community/releases.md:26]: The link is unavailable. should be `https://github.com/open-cluster-management-io/registration-operator/blob/v0.6.0/CHANGELOG/CHANGELOG-v0.6.md`?

Comment (qiujian16) [reply content/en/community/releases.md]: thanks for reminding. fixed

**Review by yue9944882 — COMMENTED** [review]
LGTM

**Review by mikeshng — APPROVED** [review]
/lgtm

File: content/en/community/releases.md [hunk 6]
```diff
@@ -5,7 +5,37 @@ weight: -20
 
 Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.5.0`. Continue reading to view upcoming releases:
 
-## `v0.6.0`, on 21st, Janurary 2022
+## `v0.6.0`, on 21st, January 2022
+
+Open Cluster Management team is proud to announce the release of OCM v0.6.0! We made many enhancements on core components and introduced some new addons.
```
Comment (juliana-hsu) [inline content/en/community/releases.md:10]: The Open Cluster Management team ...

File: content/en/community/releases.md [hunk 7]
```diff
@@ -5,7 +5,37 @@ weight: -20
...
+
+- [First release of cluster-proxy addon](https://github.com/open-cluster-management-io/cluster-proxy)
+- [First release of managed-serviceaccount addon](https://github.com/open-cluster-management-io/managed-serviceaccount)
+- [Sync status of applied resources in ManifestWork](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/29-manifestwork-status-feedback)
+- [Placement extensible scheduling](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/32-extensiblescheduling)
+- [Helm chart interface for addon framework](https://github.com/open-cluster-management-io/addon-framework/pull/62)
+- [Placement API support for multicloud-operators-subscription](https://github.com/open-cluster-management-io/multicloud-operators-subscription/pull/55)
+
+We also added many new functions in [clusteradm](https://github.com/open-cluster-management-io/clusteradm) and enhancements on documentation in the website.
```
Comment (juliana-hsu) [inline content/en/community/releases.md:19]: ... and enhanced the website documentation.

File: content/en/community/releases.md [hunk 8]
```diff
@@ -5,7 +5,37 @@ weight: -20
...
+
+To see details of the changelogs in this release:
```
Comment (juliana-hsu) [inline content/en/community/releases.md:21]: See details in the release changelogs:

**Comment by mdelder** [comment]
So many great enhancements! 

Are there docs for the new features beyond the repo READMEs (for the cluster proxy and `ManagedServiceAccount`)? Would it be helpful to link to those docs/getting started pages if they exist?

**Comment by yue9944882** [comment]
@mdelder the doc for cluster-proxy and managed-serviceaccount is already available at:

https://open-cluster-management.io/getting-started/integration/cluster-proxy/
https://open-cluster-management.io/getting-started/integration/managed-serviceaccount/

additionally there's a tutorial for plumbing up these addons together:

https://open-cluster-management.io/scenarios/pushing-kube-api-requests/

**Comment by qiujian16** [comment]
> So many great enhancements!
> 
> Are there docs for the new features beyond the repo READMEs (for the cluster proxy and `ManagedServiceAccount`)? Would it be helpful to link to those docs/getting started pages if they exist?

Thanks, added some overview with each key feature and the links to relating docs/examples.

**Pull request merged**